docs/guides/getting-started.md

```diff
@@ -49,17 +55,20 @@ setupWallet();
   <TabItem value="etherjs" label="ether.js" default>
 
 ```javascript
-const ethers = require("ethers");
+const ethers = require('ethers');
```

Comment:
Can we stick to import statements rather than require to keep things consistent?